### ISSUE TITLE
'DeprecationWarning: no_delay option is deprecated' raised even if 'no_delay' argument was not provided

### ISSUE THREAD

**Issue by khlyestovillarion** [aio-libs/aiomysql#40]
After update to aiomysql 0.0.5 I've receive such warning:
'/env/lib/python3.5/site-packages/aiomysql/connection.py:154: DeprecationWarning: no_delay option is deprecated'
It was very strange because I've not provide such argument anywhere.

After diving deep a little bit I've discovered that such warning was added in this commit - [link to commit](https://github.com/aio-libs/aiomysql/commit/364aa6d7618b4a83ff96c77806ee797d181a4602#diff-b4987acea49665e9f985c0da9f5604f4)

There on line 147 was added restriction
```
        if no_delay is not None:
            warnings.warn("no_delay option is deprecated", DeprecationWarning)
            no_delay = bool(no_delay)
        else:
            no_delay = True
```
But at the same time default argument at __init__ is `no_delay=False`.
So with that settings it raise DeprecationWarning any time you create some connection.
I'd like to fix it, but I cannot figure out what right default 'no_delay' value should be.

Versions:
aiomysql: 0.0.5
python: 3.5.0
MariaDB: mysql  Ver 15.1 Distrib 10.1.8-MariaDB
Ubuntu 14.04

**Comment by jettify** [comment]
Thanks for report! Default value should be same as in pymysql. Would you
like to provide pull request?

**Comment by khlyestovillarion** [comment]
@jettify Yes, I will fix this small issue and send a pull request

**Comment by khlyestovillarion** [comment]
https://github.com/aio-libs/aiomysql/pull/41
sorry, I don't know how to add link to the comment correct way

**Comment by jettify** [comment]
resolved in #41
